### DISCUSSION TITLE
How do I register for mobile to mobile link?

### DISCUSSION THREAD

**Discussion by wonhyunkwon** [WalletConnect/walletconnect-monorepo#1407] [Developer Support] [answered]
First of all, thanks to walletconnect for quick help. I checked the registration of our app on the Walletconnect web page. However, the purpose of our registration is to receive mobile to mobile link support. As shown in the image below, we would like to receive mobile to mobile link support for Keywallet touch, so how can I request it? I'm ready to accept even a paid service.

![스크린샷 2022-09-02 오전 10 49 50](https://user-images.githubusercontent.com/93938741/188042971-4ee9ab4e-6976-4bb5-b129-0677ecdb4b8f.png)

**Comment by chadyj** [comment, marked as answer]
For your wallet to appear in the mobile modal, simply add mobile universal or deep links to your explorer listing.

<img width="969" alt="Screenshot 2022-09-02 at 08 07 01" src="https://user-images.githubusercontent.com/116099/188069862-695f66aa-c5ea-480a-ae31-9cca7016ba95.png">
